draft-ietf-quic-qlog-quic-events.md

```diff
@@ -1522,17 +1517,34 @@ KeyType =
 
 ## QUIC Frames
 
+The generic `$QuicFrame` is defined here as a CDDL extension point (a "socket"
+or "plug"). This allows other documents to extend this extension point when
+defining new QUIC protocol frame types to enable automated validation of
+aggregated qlog schemas.
+
 ~~~ cddl
-QuicFrame =
+; The QuicFrame is any key-value map (e.g., JSON object)
+$QuicFrame /= {
+    * text => any
+}
+~~~
+{: #quicframe-def title="QuicFrame plug definition"}
+
+The QUIC frame types defined in this document are as follows:
+
+~~~ cddl
+QuicBaseFrames /=
   PaddingFrame / PingFrame / AckFrame / ResetStreamFrame /
   StopSendingFrame / CryptoFrame / NewTokenFrame / StreamFrame /
   MaxDataFrame / MaxStreamDataFrame / MaxStreamsFrame /
   DataBlockedFrame / StreamDataBlockedFrame / StreamsBlockedFrame /
   NewConnectionIDFrame / RetireConnectionIDFrame /
   PathChallengeFrame / PathResponseFrame / ConnectionCloseFrame /
   HandshakeDoneFrame / UnknownFrame
+
+$QuicFrame /= QuicBaseFrames
 ~~~
-{: #quicframe-def title="QuicFrame definition"}
+{: #quicbaseframe-def title="QuicFrames defined in this document"}
```

Comment:
I prefer the old title, its already clear the things in this document are defined in this document.